examples/hipaa/.terraform.lock.hcl

```diff
@@ -0,0 +1,114 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
I don't think we want to commit this.

Reply:
992beed2a3c5fa4be3cf82b40995695f98bce3a5